{{cookiecutter.repo_name}}/tox.ini

```diff
@@ -1,9 +1,32 @@
 [tox]
-envlist = py27, py33, py34, py35
+envlist ={% if '1.7' in cookiecutter.django_versions %}
+    py27-django17
```

Comment:
I'll make a PR myself later, but this and further sections can be consolidated a bit with `py{27,32,33,34}-django17`.